### ISSUE TITLE
Split existing time record entry into multiple

### ISSUE THREAD

**Issue by Crumar** [kimai/kimai#5002]
### Describe the problem

Sometimes, I forgot to start the next activity, e.g. when hopping between meetings.
Then I have to change the start / end times and create additional activities accordingly.
It will look like this

| CustomerProjectActivity | Start | End |
|-|-|-|
| X | 10:00 | 12:00 | 

When it should look like this:

| CustomerProjectActivity | Start | End |
|-|-|-|
| X | 10:00 | 10:30 | 
| Y | 10:30 | 11:30 | 
| Z | 11:30 | 12:00 | 

Doing this requires me to change the times, then save, add new task, save, add new task, save, ...

### Describe the solution you'd like

An context menu action to split a given entry with different assigned parameters.
Add split here:
![image](https://github.com/user-attachments/assets/bb62991b-f089-42b8-98c5-ca02e8aa982e)

A new modal / window opens, where I can add entries arbritarily and moving the times moves adjacent times as well.
Optionally, I can also add gap times, e.g. when I made a short break between activities, e.g. :
| CustomerProjectActivity | Start | End |
|-|-|-|
| X | 10:00 | 10:30 | 
| Y | 10:30 | 11:00 | 
| Z | 11:30 | 12:00 |

**Comment by christian-lo** [comment]
Yes, very helpful! I often run into the situation, that my employees track times as a block and for billing I need to separate these into multiple timesheets for each customer.
Implementation highly appreciated!

**Comment by EduNetEurope** [comment]
For us, this feature would be super helpful as well.
